codestyle.cfg

```diff
@@ -0,0 +1,231 @@
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
I tried running uncrustify with the same configuration on the test suites (`test/suites/*.function`). It messes up test framework template markers in `helpers.function`, `host_test.function` and `main_test.function` (dollar followed by an identifier: uncrustify adds a space after the `$`, and also indents but that doesn't matter). We'll have to either get uncrustify not to touch these or get the test framework to recognize different markers.